RevenueCatUI/CustomerCenter/Data/CustomerCenterConfigData.swift

```diff
@@ -1,8 +1,63 @@
 //
-//  File.swift
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+// CustomerCenterConfigData.swift
 //  
 //
-//  Created by Cesar de la Vega on 17/6/24.
+//  Created by Cesar de la Vega on 28/5/24.
 //
 
 import Foundation
+import RevenueCat
+
+struct CustomerCenterConfigData {
+
+    let id: String
+    let paths: [HelpPath]
+    let title: String
+
+    enum HelpPathType: String {
+        case missingPurchase = "MISSING_PURCHASE"
+        case refundRequest = "REFUND_REQUEST"
+        case changePlans = "CHANGE_PLANS"
+        case cancel = "CANCEL"
```

Comment:
Oh I was using lowercase for these in the backend... I'm ok changing them to uppercase though 👍